src/kvstore/LogEncoder.h

```diff
@@ -141,10 +142,7 @@ HostAddr decodeHost(LogType type, const folly::StringPiece& encoded);
  */
 int64_t getTimestamp(const folly::StringPiece& log);
 
-/**
- * @brief A wrapper class of batchs of log, support put/remove/removeRange
- */
-class BatchHolder : public nebula::cpp::NonCopyable, public nebula::cpp::NonMovable {
+class BatchHolder : public boost::noncopyable, public nebula::cpp::NonMovable {
```

Comment:
It seems that the comment of this class should not be deleted.

Reply:
has reverted